docs/openapi-issuer.yaml

```diff
@@ -0,0 +1,627 @@
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/RequestSignatureBody"
+      responses:
+        "201":
+          description: The Signature Request detail
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SignatureRequestDetailView"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "429":
+          $ref: "#/components/responses/TooManyRequests"
+        default:
+          $ref: "#/components/responses/Unexpected"
+
+  /signature-requests/{id}:
+    get:
+      operationId: getSignatureRequest
+      tags:
+        - Signature Request
+      summary: Get a Signature Request by Id
+      parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+            $ref: "#/components/schemas/Id"
+      responses:
+        "200":
+          description: The Signature Request detail
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SignatureRequestDetailView"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "429":
+          $ref: "#/components/responses/TooManyRequests"
+        default:
+          $ref: "#/components/responses/Unexpected"
+
+  /signature-requests/{id}/status:
+    put:
+      operationId: putSignatureRequestStatus
+      tags:
+        - Signature Request
+      summary: Set the status of a Signature Request
+      parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+            $ref: "#/components/schemas/Id"
+      responses:
+        "204":
+          description: Signature Request status successfully set
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "404":
+          $ref: "#/components/responses/NotFound"
+        "429":
+          $ref: "#/components/responses/TooManyRequests"
+        default:
+          $ref: "#/components/responses/Unexpected"
+
+  /signature-requests/{req_id}/documents/{doc_id}/upload-url:
+    get:
+      operationId: getDocumentUploadUrl
+      tags:
+        - Signature Request
+      summary: Get the Upload Url for the specified document
+      parameters:
+        - in: path
+          name: req_id
+          required: true
+          schema:
+            $ref: "#/components/schemas/Id"
+        - in: path
+          name: doc_id
+          required: true
+          schema:
+            $ref: "#/components/schemas/Id"
```

Comment:
Isn't it better to specify?
```
req_id -> signature_request_id
doc_id -> document_id
```

Reply:
Given the rest of the used path, I think their "meaning" is clear